src/main/java/com/dev/cinema/service/mapper/CinemaHallMapperImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.dev.cinema.service.mapper;
+
+import com.dev.cinema.model.CinemaHall;
+import com.dev.cinema.model.dto.CinemaHallRequestDto;
+import com.dev.cinema.model.dto.CinemaHallResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CinemaHallMapperImpl {
```

Comment:
```suggestion
public class CinemaHallMapper {
```
same for all mappers